lib/matplotlib/backends/web_backend/nbagg_uat.ipynb

```diff
@@ -438,10 +471,113 @@
      ],
      "language": "python",
      "metadata": {},
-     "outputs": []
+     "outputs": [],
+     "prompt_number": null
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "### UAT 17 - Timers\n",
+      "\n",
+      "Single-shot timers follow a completely different code path in the nbagg backend than regular timers (such as those used in the animation example above.)  The next set of tests ensures that both \"regular\" and \"single-shot\" timers work properly.\n",
+      "\n",
+      "The following should show a simple clock that updates twice a second:"
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": true,
+     "input": [
+      "import time\n",
+      "\n",
+      "fig, ax = plt.subplots()\n",
+      "text = ax.text(0.5, 0.5, '', ha='center')\n",
+      "\n",
+      "def update(text):\n",
+      "    text.set(text=time.ctime())\n",
+      "    text.axes.figure.canvas.draw()\n",
+      "    \n",
+      "timer = fig.canvas.new_timer(500, [(update, [text], {})])\n",
+      "timer.start()\n",
+      "plt.show()"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": [],
+     "prompt_number": null
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "However, the following should only update once and then stop:"
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": true,
+     "input": [
+      "fig, ax = plt.subplots()\n",
+      "text = ax.text(0.5, 0.5, '', ha='center') \n",
+      "timer = fig.canvas.new_timer(500, [(update, [text], {})])\n",
+      "\n",
+      "time.single_shot = True\n",
```

Comment:
Woops! I'd fixed that in the notebook I was running but forgot to save it before committing. Thanks for catching that!